templates/ts-apollo-mongodb-backend/src/index.ts

```diff
@@ -1,23 +1,40 @@
 // eslint-disable-next-line @typescript-eslint/no-require-imports
 require('dotenv').config()
+import path from 'path'
 import { ApolloServer } from "apollo-server-express"
+import { buildGraphbackAPI } from 'graphback'
+import { loadSchemaSync } from '@graphql-tools/load'
+import { GraphQLFileLoader } from '@graphql-tools/graphql-file-loader'
+import { createMongoDbProvider } from '@graphback/runtime-mongo'
 import cors from "cors"
 import express from "express"
 import http from "http"
-import { printSchema } from 'graphql'
-import { createRuntime } from './runtime'
+import { connectDB } from './db'
 
 async function start() {
   const app = express()
 
   app.use(cors())
 
-  // connect to db
-  const { schema, resolvers } = await createRuntime();
+  const modelDefs = loadSchemaSync(path.resolve('./model/*.graphql'), {
+    loaders: [
```

Comment:
I think file loader is default one. need to check if we can avoid specifying this

Reply:
The options object was at least required, I did not check if `loadSchemaSync(dir, {})` would work but without second arg it didnt work.

Reply:
![Screenshot from 2020-06-05 14-10-54](https://user-images.githubusercontent.com/11743717/83879988-6d2d9d80-a736-11ea-8ea4-1a3ea92f33b7.png)

Needs to specify it (although their docs say differently).